ui/src/app/views/recent-work/component.css

```diff
@@ -1,7 +1,9 @@
 .card-area {
-  display: inline-block;
-  width: 40%;
-  margin-left: 1%;
+    margin-left: 2.3rem;
```

Comment:
Nit: Spacing. Should be two spaces rather than 4. (If you would like I can help you configure intellij for that)

Reply:
got it 👍